recipes/flann/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set commit = "40d63fc7db5d1f6dd67364f24e4579aedb57e7b6" %}
+
+{% set tag = environ.get("GIT_DESCRIBE_TAG", "") %}
+{% set number = environ.get("GIT_DESCRIBE_NUMBER", "0") %}
+{% set hash = environ.get("GIT_DESCRIBE_HASH", "0") %}
+{% set tag_ext = ("+" + number + "." + hash) if number != "0" else "" %}
+{% set version = tag + tag_ext %}
+
+
+package:
+    name: flann
+    version: {{ version }}
+
+source:
+    git_url: https://github.com/mariusmuja/flann.git
+    git_tag: {{ commit }}
+
+build:
+    number: 0
+    skip: True    # [win]
+
+requirements:
+    build:
+        - gcc     # [osx]
+        - hdf5
+        - cmake
+    run:
+        - libgcc  # [osx]
+        - hdf5
+
+#test:
+#    commands:
+#        # Verify libraries.
+#        {% set flann_libs = ["flann", "flann_s", "flann_cpp", "flann_cpp_s"] %}
+#        {% for each_flann_lib in flann_libs %}
+#        - test -f $PREFIX/lib/lib{{ each_flann_lib }}.a                           # [unix]
+#        - test -f $PREFIX/lib/lib{{ each_flann_lib }}.dylib                       # [osx]
+#        - test -f $PREFIX/lib/lib{{ each_flann_lib }}.so                          # [linux]
+#        - if not exist %PREFIX%\\Library\\lib\\{{ each_flann_lib }}.lib exit 1    # [win]
+#        - if not exist %PREFIX%\\Library\\bin\\{{ each_flann_lib }}.dll exit 1    # [win]
+#        {% endfor %}
```

Comment:
Is there some reason this is commented? Is there something we can help with here?